common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationSupport.java

```diff
@@ -143,6 +145,36 @@ public static boolean execICU(final UTF8String l, final UTF8String r,
    * Collation-aware regexp expressions.
    */
 
+  public static class StringSplit {
+    public static UTF8String[] exec(final UTF8String l, final UTF8String r, final int limit,
```

Comment:
let's be consistent with naming

`string, regex, limit` instead of `l, r, limit`